OpenRA.Mods.Common/Scripting/Global/UtilsGlobal.cs

```diff
@@ -98,6 +98,12 @@ public int RandomInteger(int low, int high)
 			return Context.World.SharedRandom.Next(low, high);
 		}
 
+		[Desc("Returns a random floating point value.")]
```

Comment:
I think I will better remove it then. It was just used as a test case where I created an epileptic disco world and understand how Lua works with floats.